cnd/src/http_api/hbit_herc20/alice.rs

```diff
@@ -40,7 +40,10 @@ impl RedeemAction
 {
     type Output = ethereum::CallContract;
 
-    fn redeem_action(&self) -> anyhow::Result<Self::Output> {
+    fn redeem_action(
+        &self,
+        _bitcoin_fee_rate_per_vbyte: bitcoin::Amount,
```

Comment:
Leaving as `_` might bring it onto one line again :)

```suggestion
        _: bitcoin::Amount,
```